dependencies/scripts/mongo-init-replica-set.sh

```diff
@@ -3,7 +3,7 @@
 mongo <<EOF
 var config = {
     _id: "heron_rs",
-    members: [{ _id: 0, host: "mongo:27017"}]
+    members: [{ _id: 0, host: "127.0.0.1:27017"}]
```

Comment:
You'll likely find that changing this stops the Docker stack from initialising the replica set properly.

Reply:
Actually no I'm wrong.  The port is exposed to the local host.  It has been working for me in the previous format though so I am not sure why you've needed to change it

Reply:
I am running the services in docker container but tests locally. The hostname mongo cannot be resolved. Changing hostname to loopback seemed harmless as the port is exposed as you said.